### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AnastasiiaHombalevska**
🦊 [DEMO LINK](https://AnastasiiaHombalevska.github.io/react_autocomplete/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import classNames from 'classnames';
+import { Person } from './types/Person';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+export const App: React.FC<Person | {}> = () => {
+  const [query, setQuery] = useState('');
+  // const [title, setTitle] = useState('No selected person');
```
Comment (DarkMistyRoom) [inline src/App.tsx:9]: remove all comments

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
- should set the title to "No selected person" after input is changed. Take this page as an example https://mykhailonl.github.io/react_autocomplete/
- also hide the dropdown list on changes and don't show the error message
- hide the dropdown after selecting a person

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great progress! Keep doing! A user should be able to select a person

**Review by ghost — APPROVED** [review]
Good job!

**Pull request closed without merging**